paymentsheet/src/main/java/com/stripe/android/paymentsheet/viewmodels/BaseSheetViewModel.kt

```diff
@@ -561,8 +562,13 @@ internal abstract class BaseSheetViewModel(
         ((paymentMethodMetadata.value?.stripeIntent as? PaymentIntent)?.requireCvcRecollection == true) &&
             FeatureFlags.cvcRecollection.isEnabled
 
+    internal fun isCvcRecollectionForDeferred(): Boolean {
+        return CvcRecollectionCallbackHandler.isCvcRecollectionEnabledForDeferredIntent() &&
+            FeatureFlags.cvcRecollection.isEnabled
+    }
+
     internal fun getCvcRecollectionState(): PaymentSheetScreen.SelectSavedPaymentMethods.CvcRecollectionState {
-        return if (isCvcRecollectionEnabled()) {
+        return if ((isCvcRecollectionEnabled() || isCvcRecollectionForDeferred())) {
```

Comment:
Are there tests for this?

Reply:
I have [ticket](https://jira.corp.stripe.com/browse/MOBILESDK-2220) for writing tests here. Doing separately so we can start bug bash

Reply:
Done 👍